jupyterlab_ucaip/src/components/datasets/dataset_component.tsx

```diff
@@ -254,6 +338,30 @@ ${this.getColumnString()} }
                   </Select>
                   <FormHelperText>Objective</FormHelperText>
                 </FormControl>
+                <TextField
+                  style={{ paddingLeft: '24px', paddingTop: '3px', width: 50 }}
+                  error={this.state.budgetError}
+                  onChange={event => {
+                    let val = event.target.value;
```

Comment:
```val``` seems like it could be a more descriptive name. Its brevity makes this section harder to follow.